### DISCUSSION TITLE
SubAPI staking ranking #1

### DISCUSSION THREAD

**Discussion by jiguantong** [ringecosystem/discussion#3] [General]
https://github.com/orgs/subapidao/discussions/48

- [x] 2/26: Confirm ORMP Oracle signers & staking
- [x] 2/27: Transfer staking RING to SubAPIDAO(`0x000000000a0D8ac9cc6CbD817fA77090322FF29d`), [SLA](https://github.com/orgs/subapidao/discussions/48)
- [x] 2/29: ORMP Oracle operator run node

For example:
0x9F33a4809aA708d7a399fedBa514e0A0d15EfA85 staking commit: 30,000 RING

**Comment by jiguantong** [comment]
@hackfisher 0x00000052b2A5b4d02e45C5f9F0C8B7E62FEbD08E   staking commit: 25,000 RING

**Comment by jiguantong** [comment]
@jiguantong 0x9F33a4809aA708d7a399fedBa514e0A0d15EfA85 staking commit: 30,000 RING

**Comment by wuminzhe** [comment]
@wuminzhe 0xA4bE619E8C0E3889f5fA28bb0393A4862Cad35ad staking commit: 30,000 RING

**Comment by casinoyoda** [comment]
@casinoyoda 0xFa5727bE643dba6599fC7F812fE60dA3264A8205 staking commit: 30,000 RING

**Comment by fewensa** [comment]
0x178e699c9a6bb2cd624557fbd85ed219e6faba77 staking commit: 30,000 RING

**Comment by hujw77** [comment]
@hujw77 `0xB9a0CaDD13C5d534b034d878b2fcA9E5a6e1e3A4` staking commit: 30,000 RING

**Comment by jiguantong** [comment]
Result:
@jiguantong 0x9F33a4809aA708d7a399fedBa514e0A0d15EfA85 staking commit: 30,000 RING
@wuminzhe 0xA4bE619E8C0E3889f5fA28bb0393A4862Cad35ad staking commit: 30,000 RING
@casinoyoda 0xFa5727bE643dba6599fC7F812fE60dA3264A8205 staking commit: 30,000 RING
@fewensa  0x178e699c9a6bb2cd624557fbd85ed219e6faba77 staking commit: 30,000 RING
@hujw77 0xB9a0CaDD13C5d534b034d878b2fcA9E5a6e1e3A4 staking commit: 30,000 RING

Please transfer your staking to SubAPIDAO: 0x000000000a0D8ac9cc6CbD817fA77090322FF29d, and put the transaction hash here.

**Reply by jiguantong** [reply]
@jiguantong https://darwinia.subscan.io/tx/0x141b8876a5500ee3a4f7863bcfcf9d2afca1b073a479648489f32034494eaa61

**Reply by wuminzhe** [reply]
@wuminzhe https://darwinia.subscan.io/tx/0xb403e867f6a96a1b8843a525143f33867d876b15c719abe17bf139d5b8c91c2c

**Reply by hujw77** [reply]
@hujw77 https://darwinia.subscan.io/tx/0x7dcdc2d7e4a763fa791f4ef3e73a2fa524c6e7efd63b43df9c07dc649cf94070

**Reply by fewensa** [reply]
@fewensa https://darwinia.subscan.io/tx/0xd5a48eda61bbf998e432f612d83875fc369e3eaf970475cef425f43e0449f557

**Reply by casinoyoda** [reply]
@casinoyoda https://darwinia.subscan.io/tx/0x6474aec02d77ab3e8be35ab1397272c62fb289c170c7f0dcc48a32c5ebf9aafd
